docs/guide/server.md

```
@@ -0,0 +1,634 @@
+  children: SpatialNode[];
+  elevation?: number;
+}
+```
+
+## Parquet Format
+
+The server uses Apache Parquet for efficient binary serialization.
+
+### Standard Format (15x Smaller)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)